aerospike-proxy/examples/kind/aerospike.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: asdb.aerospike.com/v1
+kind: AerospikeCluster
+metadata:
+  name: aerospike-cluster
+  namespace: aerospike
+spec:
+  size: 1
+  image: aerospike/aerospike-server-enterprise:7.0.0.5
```

Comment:
I assume this is for testing purposes? if so we might want to use Aerospike Database 7.1+ to be able to test some of the latest features such as LRU functionality (`readTouchTtlPercent`)

Reply:
sure! will change